esp/scm/WsDeploy.ecm

```diff
@@ -568,7 +568,7 @@ ESPresponse [exceptions_inline, encode(0)] GetSummaryResponse
     string XmlStr;
 };
 
-ESPservice [exceptions_inline("xslt/exceptions.xslt")] WsDeploy
+ESPservice [requiredaccess("NONE"), exceptions_inline("xslt/exceptions.xslt")] WsDeploy
```

Comment:
Saying NONE at this level is kind of confusing, because in fact it still does service level authentication, right?  That is it still prompts me for a valid user id and password?  Somehow seems like we need to make clear the difference between accessing the process, the service, and the features.  It may be as simple as renaming this attribute.

It currently sounds like "service required access" but you are really talking about "default feature level access".

Up for discussion, but maybe something like everywhere that you currently have "requiredaccess", change to "featureaccess" to be less confusing.